docs/server-dependencies.md

```diff
@@ -138,7 +154,7 @@ Refer to this [Makefile](https://github.com/temporalio/temporal/blob/v1.3.2/Make
 	--ep localhost -p 3036 -u root -pw root --pl mysql --db temporal_visibility update-schema -d ./schema/mysql/v57/visibility/versioned/
```

Comment:
where do i find the docs for `temporal-cassandra-tool`? shall we link people to it

Reply:
None yet AFAIK -

Reply:
yikes